src/test/java/io/pravega/connectors/flink/FlinkPravegaTableITCase.java

```diff
@@ -310,19 +325,20 @@ private void testTableSourceBatchDescriptor(Stream stream, PravegaConfig pravega
                 .withSchema(schema);
 
         final Map<String, String> propertiesMap = desc.toProperties();
-        final TableSource<?> source = TableFactoryService.find(BatchTableSourceFactory.class, propertiesMap)
```

Comment:
We should not change the batch test now. We should keep it as is and uncomment the test after fixing #341